lib/components/EmailInput.js

```diff
@@ -92,7 +96,18 @@ const EmailInput = ({
       <div>
         <Typography style="body3" className="neeto-ui-text-error mt-1">
           {!!error && Array.isArray(error)
-            ? "Please make sure all emails are valid."
+            ?
+            <div className="flex">
+              {"Please make sure all emails are valid."}
+              <Typography
+                className="cursor-pointer"
+                style="body3"
+                weight="bold"
+                onClick={handleFilterEmails}
+              >
+                &nbsp;Click here to remove invalid emails.
```

Comment:
Let's allow this upon a prop. @karthiknmenon what's your thought on providing the option to customize this text?

Reply:
@amaldinesh7 Yes. I agree. 

cc: @Thejus-Paul